### ISSUE TITLE
I suppose there are some remains after I delete an Upload at UI

### ISSUE THREAD

**Issue by W-Park** [fossology/fossology#1090]
Hi,
On my FOSSOlogy 3.3.0 system, 
When I upload a tar file and delete the Upload at Web UI, I suppose there are some remains in the repository directory after deleting it.
(Actually I think same thing happened in 3.2.0 and 3.0.0 aswell.)
![repository](https://user-images.githubusercontent.com/33045921/39615233-18e8847c-4fb0-11e8-9e22-9590e755b81c.png)

Is it common thing to happen?
Or is it related the log (as followes)

> 2018-05-04 15:22:27 delagent [0] :: JOB[363].delagent[29967.localhost]: "NOTICE:  table "delup_43_pfil    e" does not exist, skipping"

**Comment by shaheemazmalmmd** [comment]
@JustinWonjaePark : Yes there will be some data remaining in even after the upload is deleted.
